policies/rudderc/templates/technique.ps1.askama

```diff
@@ -1,19 +1,23 @@
 {#- Syntax: https://djc.github.io/askama/template_syntax.html -#}
-function {{ id }}
-{
+function {{ id }} {
     [CmdletBinding()]
     param (
         [parameter(Mandatory = $true)]
         [string]$reportId,
         [parameter(Mandatory = $true)]
         [string]$techniqueName,
 {% for p in parameters %}
-        [parameter(Mandatory = $true)]
-        [{{ p._type }}]${{ p.name }},
+        [parameter(Mandatory = ${{ !p.may_be_empty }})]
```

Comment:
I do not think we ever really tested the non-mandatory technique parameters on windows....
We need to double check that it can supports empty inputs, depending on the directive generation we may need to add the  `[AllowEmptyString()]` flag on the non mandatory ones.

Reply:
I really try to stick with what we currently generate, but I'd be happy to fix it if we find it incorrect.